### ISSUE TITLE
Allow user to specify hotkeys using strings rather than raw keycodes

### ISSUE THREAD

**Issue by neithernut** [waysome/waysome#583]
ATM, hotkeys have to be specified raw in the API. It would be nice if we could specify them via strings, like "Ctrl", "Alt", "Q".
This is a relatively elaborate task:
- First of all, the asignee has to investigate how to translate a string to a keycode properly. He/she may then have to write a function to perform this task.
- The existing method of the global context has to be adapted. The functionality of setting the key-codes directly may be preserved.
- The default config file should be adjusted to use this new functionality.

**Comment by TheNeikos** [comment]
I don't think that translating it to the locale of the user is a good idea.

Reasons for that:
- Other locales have weird buttons, sometimes which are hard to enter or process. (And don't look good in a config)
- It is easy to ship a program similar to `xev` that will output the keycodes
- It makes it easy for us to implement it (which is a huge plus)

**Comment by neithernut** [comment]
All those points are valid, but still, the hotkeys are not very readable the way they are presented now.
If it wouldn't confuse user's I had suggested supporting representation of keycodes as key names as present on an standart US PC keyboard (e.g. as in `linux/input.h`).

**Comment by TheNeikos** [comment]
Yup, I forgot to mention that I envisioned it like that.

**Comment by matthiasbeyer** [comment]
I agree with you guys, but I think this is not important or should be solved within the project phase. It's simply a convenience thing, so I'd suggest to put this on hold until we have time for it. There is more important stuff to do!

**Comment by neithernut** [comment]
Well, you said to me something like "spam issues so I have something to do". So don't complain if they are not of a high priority.

**Comment by matthiasbeyer** [comment]
@TheNeikos said that with "spam", but yes...
